xmodule/tests/test_library_tools.py

```diff
@@ -63,16 +66,16 @@ def test_import_from_blockstore(self):
         course = CourseFactory.create(modulestore=self.store, user_id=self.user.id)
         CourseInstructorRole(course.id).add_users(self.user)
         # Add Source from library block to the course
-        sourced_block = self.make_block("library_sourced", course, user_id=self.user.id)
+        lc_block = self.make_block("library_content", course, user_id=self.user.id)
```

Comment:
This change (library_sourced being deleted in favor of library_content) happened in the source code few weeks ago. The fact that this test case was still passing (under LMS settings) means that it didn't really work!

When run under CMS settings, though, the test case did fail (like it should have).